Tasks/GooglePlayPromote/google-play-promote.ts

```diff
@@ -53,8 +54,13 @@ async function run() {
         let track = await googleutil.getTrack(edits, packageName, sourceTrack);
         tl.debug(`Current track: ${JSON.stringify(track)}`);
 
+        let versionNumber = track.releases[0].versionCodes;
+        if (versionCode !== undefined) {
+            versionNumber = [Number(versionCode)];
```

Comment:
Could it make sense to show some message for the case of invalid value (not a positive integer - according to [docs](https://developer.android.com/studio/publish/versioning))?

Reply:
I am not sure if adding a message here would be helpful.
Since I seem to not be able to detect which version codes are currently available for promotion on the source track, I am not sure if a check in the plugin would make sense. Currently I only get the latest release from soruce track via the API call already implemented.
However a check is done by google on their side, if e.g. a version code is not available.
e.g. I would get the following error, if version code 500 is not available on source track:
"Error: The following APK version codes could not be found: [500]."
I have not tested negative version codes and the max version code, but I would expect a similar error message.

While we are at it, I don't think my added test makes much sense, if version codes are checked by google.
I probably have to check if the correct version code are sent if one is provided vs not provided.
I will take a look later today and update the pull request on the tests